### PR TITLE
[incubator/janitor] Add hjacobs/kube-janitor

### PR DESCRIPTION
**Pull request by mayppong**
There is already another chart named kube-janitor by another person so
the name for this chart has been changed to just janitor as OK'ed by the
developer. See (https://github.com/hjacobs/kube-janitor/issues/44)

#### Is this a new chart
Yes.

#### What this PR does / why we need it:
Adds another new chart to the repo

#### Which issue this PR fixes
n/a

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: incubator/janitor/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+apiVersion: v1
+appVersion: "v19.9.0"
+description: A Helm chart for running hjacobs/kube-janitor
```
Comment (hjacobs) [inline incubator/janitor/Chart.yaml]: I think the description should be more meaningful like:

```
Clean up (delete) Kubernetes resources after a configured TTL (time to live) 
```

File: incubator/janitor/README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
+## Configuration
```
Comment (hjacobs) [inline incubator/janitor/README.md]: the README needs some proper content

**Comment by scottrigby** [comment]
Hi 👋 new charts are no longer accepted to `stable` or `incubator`. See [Status of the Project](https://github.com/helm/charts#status-of-the-project) and [Deprecation Timeline](https://github.com/helm/charts#deprecation-timeline) for more info about that.

Good news is you can host a Helm chart yourself on GitHub, use the same tools as this repo does for automatic Chart [testing](https://github.com/helm/chart-testing) and [releasing](https://github.com/helm/chart-releaser), and list your Helm repo on https://helm.hub.sh. Follow this issue for info about existing guidelines and upcoming improvements: https://github.com/helm/charts/issues/7800

**Pull request closed without merging**